common4j/src/main/com/microsoft/identity/common/java/flighting/CommonFlight.java

```diff
@@ -109,7 +109,7 @@ public enum CommonFlight implements IFlightConfig {
     /**
      * Flight to enable adding x-client-MN and x-client-WPAvailable extra query parameters
      */
-    ENABLE_AM_API_WORKPROFILE_EXTRA_QUERY_PARAMETERS("EnableAmApiWorkProfileExtraQueryParameters", false),
+    ENABLE_AM_API_WORKPROFILE_EXTRA_QUERY_PARAMETERS("EnableAmApiWorkProfileExtraQueryParameters", true),
```

Comment:
>true [](http://example.com/codeflow?start=99&length=4)

why this change?

Reply:
it should be enabled by default. This was a missed change i had made for validation purposes. Idea is we will make a PR in ecs to turn this off if we run into any issues in prod